tests/publiccloud/check_registercloudguest.pm

```diff
@@ -194,7 +200,8 @@ sub test_container_runtimes {
             record_info('permissions #1', 'permissions when libcontainers-common is missing');
             $instance->ssh_script_run('sudo stat /etc/containers/registries.conf');
             $instance->ssh_script_run('sudo rm -rf /etc/containers/registries.conf');
-            $instance->ssh_assert_script_run("sudo zypper -n install libcontainers-common");
+            # $instance->ssh_assert_script_run("sudo zypper -n install libcontainers-common");
+            $instance->zypper_call_remote("install libcontainers-common");
```

Comment:
```suggestion
            $instance->zypper_call_remote(" -n install libcontainers-common");
```

Reply:
sorry 🙂 but it is not needed, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/23960#discussion_r2518930722 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/23960#discussion_r2518926548